test/blackbox/sync_docker.bats

```diff
@@ -80,3 +84,125 @@ function teardown_file() {
     [ "$status" -eq 0 ]
     [ $(echo "${lines[-1]}" | jq '.tags[]') = '"latest"' ]
 }
+
+@test "sync image on demand for kubernetes" {
+    run skopeo copy docker://127.0.0.1:8090/kube-apiserver-amd64:v1.10.0 oci:${TEST_DATA_DIR} --src-tls-verify=false
+    [ "$status" -eq 0 ]
+
+    run curl http://127.0.0.1:8090/v2/_catalog
+    [ "$status" -eq 0 ]
+    [ $(echo "${lines[-1]}" | jq '.repositories[1]') = '"kube-apiserver-amd64"' ]
```

Comment:
'.repositories[1]' seems to be arbitrary. why `1`? How do we make sure these tests don't fail?

Can we do something like https://cloudaffaire.com/faq/how-to-check-if-a-certain-key-or-value-exists-in-json-data-using-jq/ ?
Or something like https://stackoverflow.com/questions/18592173/select-objects-based-on-value-of-variable-in-object-using-jq ?